README.md

```diff
@@ -181,6 +181,20 @@ Important Notes:
 - The key/value pairs displayed in the `old_record` field include the table's identity columns for the record being updated/deleted. To display all values in `old_record` set the replica identity for the table to full
 - When a delete occurs, the contents of `old_record` will be broadcast to all subscribers to that table so ensure that each table's replica identity only contains information that is safe to expose publicly
 
+## Error States
```

Comment:
@olirice are you planning on adding `Error 413: Payload Too Large` to this new error section?